### DISCUSSION TITLE
organizzazione fisica - 7

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#86] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/23eb8dba-ff14-4d43-8ffb-abbb76b3877b)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-22 161715](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/13480d18-3965-40b0-9530-05425f5a021b)
![Immagine 2024-01-22 161727](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/afcadb78-3815-4429-be3d-0268fd9ff260)
![Immagine 2024-01-22 161737](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/3b5f5f38-a93d-48b4-8383-37a9743bc145)

**Reply by 7Andrei** [reply]
I blocchi per bucket a me vengono 489(arrotondato per eccesso) per il resto tutto uguale

**Reply by daromacarlo** [reply]
@7Andrei ormai io e gli errori di calcolo siamo la stessa cosa

**Comment by Frisbon** [comment]
gira la testa

![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/ba6da775-fcb8-4981-b50d-4bbbd1221cbf)
